lib/srv/discovery/fetchers/aws-sync/aws-sync.go

```diff
@@ -204,42 +208,42 @@ func (a *awsFetcher) poll(ctx context.Context, features Features) (*Resources, e
 	// - attached policies
 	// - user groups they are members of
 	if features.Users {
-		eGroup.Go(a.pollAWSUsers(ctx, result, collectErr))
+		eGroup.Go(a.pollAWSUsers(ctx, result, a.lastResult, collectErr))
```

Comment:
Since here you call a receiver on `a` already, why do you need to pass `a.lastResult`?